### PR TITLE
Adds the initial view objects

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**

### CODE REVIEW

File: server/routes/createGoal/createGoalController.ts [hunk 1]
```diff
@@ -3,6 +3,9 @@ import type { Prisoner } from 'prisonRegisterApiClient'
 import type { PrisonerSummary } from 'viewModels'
 import createError from 'http-errors'
 import PrisonerSearchService from '../../services/prisonerSearchService'
+import CreateGoalView from './createGoalView'
+import AddStepView from './addStepView'
+import AddSNoteView from './addNoteView'
```
Comment (mattwills-moj-digital) [inline server/routes/createGoal/createGoalController.ts:8]: Typo

File: server/routes/createGoal/createGoalView.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    prisonerSummary: PrisonerSummary
+    form: CreateGoalForm
+    errors?: Array<Record<string, string>>
+  } {
+    return {
+      prisonerSummary: this.prisonerSummary,
+      form: this.createGoalForm,
+      errors: this.errors || [],
+    }
+  }
```
Comment (mattwills-moj-digital) [inline server/routes/createGoal/createGoalView.ts:21]: These three view objects are identical right? I don't follow why we can't use the same one, or perhaps use inheritance 🤔

File: server/routes/createGoal/createGoalController.ts (hunk 1 above)
Comment (nathanrussell-moj-digital) [reply server/routes/createGoal/createGoalController.ts:8]: Whoops! Let me correct that!

File: server/routes/createGoal/createGoalView.ts (hunk 2 above)
Comment (nathanrussell-moj-digital) [reply server/routes/createGoal/createGoalView.ts:21]: I had the same thought TBH .... lets talk it through

**Review by mattwills-moj-digital — APPROVED** [review]

**Pull request merged**